pkg/apply/applier.go

```diff
@@ -207,7 +209,12 @@ func (a *Applier) Run(ctx context.Context) <-chan event.Event {
 		}
 
 		if a.StatusOptions.wait {
-			statusChannel := a.resolver.WaitForStatusOfObjects(ctx, infosToObjects(infos))
+			statusChannel := a.statusPoller.Poll(ctx, infosToObjMetas(infos), polling.Options{
+				PollUntilCancelled: true,
+				PollInterval:       a.StatusOptions.period,
+				UseCache:           true,
+				DesiredStatus:      status.CurrentStatus,
+			}) //.WaitForStatusOfObjects(ctx, infosToObjects(infos))
```

Comment:
Should this comment be removed // .WaitForStatusOfObjects ...

Reply:
Yeah, I have removed it.